WcaOnRails/spec/controllers/api_controller_spec.rb

```diff
@@ -319,10 +321,10 @@
       before :each do
         user = FactoryBot.create :user
 
-        wrc_team = Team.find_by_friendly_id('wrc')
+        wrc_team = Team.find_by_friendly_id(Team::WRC_FRIENDLY_ID)
```

Comment:
As per my earlier suggestion, we could change this to `Team.wrc` =)

Reply:
This and all others :)